l10n_ir_account/data/template/account.account-ir.csv

```diff
@@ -384,4 +384,3 @@
 "account_account_template_0_911108","سایر اسناد تضمینی به عهده شرکت","911108","off_balance","","0"
 "account_account_template_0_911201","طرف حسابهای انتظامی به عهده شرکت","911201","off_balance","","0"
 "account_account_template_0_911202","طرف حسابهای انتظامی به نفع شرکت","911202","off_balance","","0"
```

Comment:
Hi
here name must be in English and then translated in the po, and the id could be shorter
https://github.com/OCA/l10n-iran/blob/15.0/l10n_ir/data/account.account.template.csv

Reply:
hi,this is not mandatory.
this iran chart also must be refine in future,we can do it later
https://github.com/odoo/odoo/blob/17.0/addons/l10n_ua/data/template/account.account-ua_ias.csv

now tax,tax group , fiscal position and default accounts have priority to do  first